AlertaDengue/dados/views.py

```diff
@@ -390,7 +390,7 @@ def get_context_data(self, **kwargs):
         # municipios, geocodigos = list(mundict.values()), list(mundict.keys())
         # results[d] = dbdata.load_serie_cities(geocodigos, d)
         for d in diseases:
-            case_series[d] = dbdata.get_series_by_UF(d, 52)
+            case_series[d] = dbdata.get_series_by_UF(d, 51)
```

Comment:
I guess this error is not related to the error on CI ... 
you should keep your PR as short as possible.